parts/common/components.json

```diff
@@ -983,6 +983,23 @@
         }
       }
     },
+    {
+      "name": "aks-secure-tls-bootstrap-client",
+      "downloadLocation": "/usr/local/bin",
+      "downloadURIs": {
+        "default": {
+          "current": {
+            "versionsV2": [
+              {
+                "renovateTag": "<DO_NOT_UPDATE>",
+                "latestVersion": "0.1.0-alpha.5"
+              }
+            ],
+            "downloadURL": "https://github.com/Azure/aks-secure-tls-bootstrap/releases/download/client/v${version}/linux-${CPU_ARCH}.tar.gz"
```

Comment:
this might not be available for some customer locking down their network

Reply:
we don't be downloading from github at provisioning time by the time this hits preview/GA

Reply:
we wither won't be downloading at provisioning time at all, or downloading from MCR via oras